packages/astro/playwright.config.js

```diff
@@ -24,9 +24,6 @@ const config = {
 	use: {
 		/* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
 		actionTimeout: 0,
-		/* Base URL to use in actions like `await page.goto('/')`. */
-		baseURL: process.env.PLAYWRIGHT_TEST_BASE_URL || 'http://localhost:3000',
-
```

Comment:
Remove this `baseURL` to discourage navigating `goto` without a base url / host (e.g. always use `astro.resolveUrl`). So that each test can navigate with their own unique port.